modules/tinymce/src/plugins/template/main/ts/ui/Dialog.ts

```diff
@@ -64,6 +59,11 @@ const getPreviewContent = (editor: Editor, html: string) => {
         '"' + cors + '>';
     });
 
+    const contentStyle = Settings.getContentStyle(editor);
```

Comment:
Worth adding tests for this order change, although it should be as simple as altering the existing `DialogGetPreviewContentTest.ts`

Reply:
Its failing at the moment. I'm on it right now

Reply:
Fixed.